venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -264,22 +192,16 @@ function() {
           }
         }
 
-        var groupPromises = Promise.all([
+        var batchPromises = Promise.all([
           or3client.or3request(or3client.grpUrl, authorGroup, 'POST', token),
-          or3client.or3request(or3client.grpUrl, reviewerGroup, 'POST', token),
-          or3client.or3request(or3client.grpUrl, areachairGroup, 'POST', token),
-          or3client.or3request(or3client.grpUrl, anonReviewer1Group, 'POST', token),
-          or3client.or3request(or3client.grpUrl, anonReviewer2Group, 'POST', token),
-          or3client.or3request(or3client.grpUrl, anonReviewer3Group, 'POST', token),
-          or3client.or3request(or3client.grpUrl, anonReviewer4Group, 'POST', token),
-          or3client.or3request(or3client.grpUrl, anonReviewer5Group, 'POST', token),
           or3client.or3request(or3client.inviteUrl, withdrawPaperInvitation, 'POST', token),
           or3client.or3request(or3client.inviteUrl, publicCommentInvitation, 'POST', token),
           or3client.or3request(or3client.inviteUrl, officialCommentInvitation, 'POST', token),
-          or3client.addGroupMember(AUTHORS, note.content.authorids.concat(note.signatures), token)
+          or3client.addGroupMember(AUTHORS, note.content.authorids.concat(note.signatures), token),
```

Comment:
what is this, the general group with all the authors?

Reply:
yes, having a list of all authors is useful to the program chairs for sending emails

Reply:
I imagined that. We need to be careful about this, because if the paper is updated with new authors or withdrawn then this group is outdated. 

My recommendation is to send the emails based on the author groups of the active papers.

Reply:
Or, have a job that can update this group periodically.

Reply:
I see - ok, do you think I should get rid of this then?

Reply:
Yes, to keep the process function as clean and lightweight as possible. We can create this group later, what do you think?